src/corg/coding-and-design-guidelines/code-review-guidelines.md

```diff
@@ -7,74 +7,235 @@ review:
 tree_item_index: 300
 toc: true
 ---
+
 Code reviews are a collaborative process between coders and reviewers — this is not a battle. The goal of the reviews is to improve the code quality by having several pairs of eyes on the code, for the benefit of all.
 
-## When Creating a Pull Request
+Before you read this page, make sure you've read the basic [Contribution Principals]({{page space='nxdoc' page='next/contributing-to-nuxeo'}}).
+
+These guidelines apply to the repositories of the [Nuxeo](https://github.com/nuxeo/) GitHub organization. Yet, some specific rules apply to the [Platform repositories](#platform-repositories), i.e. the ones maintained by the Platform team and built in a dedicated CI system.
+
+## When Creating a Pull Request (PR)
+
+### Use the Draft Feature
+
+Don’t hesitate to start with creating a draft PR. Then, you can ask for an early review if you feel it’s worth it, by using the GitHub [Reviewers](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review) feature.
 
-### Follow the Coding Standards
+When the draft PR sounds mature enough and you want some people to actually review it, switch it to "Ready for review".
 
+You can, of course, create a PR and ask for reviews directly.
+In any case, make sure you read your own PR before submitting it to other people.
```

Comment:
For me if the PR is ready for review, the CI should be OK before requesting reviewer

Reply:
Added a paragraph in the Platform Repositories section to mention that it is up to the reviewer to wait or not for the CI checks.